cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -84,6 +84,8 @@ spec:
             - --container=heapster
             - --poll-period=300000
             - --estimator=exponential
+          securityContext:
+            runAsNonRoot: false
```

Comment:
> They run as root, but they shouldn't.

Actually, I don't understand this statement. `runAsNonRoot: false` means they explicitly reserve the right to run as root if the image dictates it, right?